examples/downstream-v1.sh

```diff
@@ -76,7 +75,7 @@ set -x
 
 ${TOOLS_REPO_DIR}/v1  \
        --mode=synchronize \
-       --catalogd-dir=${SYNC_DIR}/operator-framework-catalogd \
+       --ignore-catalogd \
```

Comment:
This option is no longer necessary, given #65

Reply:
Done